package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "lerna:version": "yarn && lerna version && yarn run cirlce:release",
     "cirlce:release": "git tag -d RELEASE && git tag RELEASE && git push --delete origin RELEASE && git push --tags",
-    "lerna:publish": "lerna publish from-git --yes",
+    "lerna:publish": "node lerna.js",
```

Comment:
- lerna.js is not a descriptive/accurate file name/
- `publish` `publish-npm` `publish-lerna` (last is probably best).